interface/src/components/requests.js

```diff
@@ -29,7 +29,7 @@ export function getVisualData(files, type, username){
         params= 'getTreatmentSites'
     }
     
-    let res= fetch(`http://localhost:3001/${params}`,  {
+    let res= fetch(`http://localhost:80/${params}`,  {
```

Comment:
It would be better to get the URI from the runtime environment than hard coding this. This would also allow https & alternate ports to be used

Reply:
Thanks for pointing out! Will change this.